roles/ansible-os-hardening/vars/main.yml

```diff
@@ -36,15 +27,12 @@ os_security_suid_sgid_whitelist: []
 os_security_suid_sgid_remove_from_unknown: false
 
 # remove packages with known issues
-os_security_packages_clean: true
+os_security_packages_clean: false
```

Comment:
@rndmh3ro Is there any reason, why we set this value to false? I admit we do not have a great test coverage in our specs yet.